examples/basic_sg_distributed_index_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "log":[
+    "HTTP+"
+  ],
+  "adminInterface": "127.0.0.1:4985",
+  "interface":"0.0.0.0:4984",
+  "databases":{
+    "default":{
+      "feed_type":"DCPSHARD",
```

Comment:
Feed type shouldn't be specified.  (It's not used for SG nodes when running w/ accel).